storage/CompoundAction/Formatter.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2017 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+
+#include "storage/CompoundAction/Formatter.h"
+
+
+namespace storage
+{
+
+    using std::string;
+
+    CompoundAction::Formatter::Formatter(const CompoundAction::Impl* compound_action)
+    : compound_action(compound_action), tense(Tense::SIMPLE_PRESENT) {}
```

Comment:
I still don't see why any tense other than simple present should be used. Removing the variable would save lots of code and messages to translate.